client/src/elements/schema-editor/schema-editor-files.js

```diff
@@ -195,3 +210,26 @@ export class SchemaEditorFiles {
     });
   }
 }
+
+function uuid() {
```

Comment:
I just saw that there exists already a class `IdGenerator` in q-editor -> https://github.com/nzzdev/Q-editor/blob/b33b34a9ce7aa5f0bb80eaec3e45e317ef429ba0/client/src/resources/IdGenerator.js

I think it would make sense to extend this class with a method `getUUID()` and use it here. Maybe this can be useful in other parts of the editor in the future.